### PR TITLE
Allow configuring custom web3 instance

### LINKED ISSUE #35: Using the helpers outside of Truffle (i.e. remove assumption of global web3)
I am trying to `const oz = require('openzeppelin-test-helpers')` but I am getting the following error:

```
/path/to/project/node_modules/openzeppelin-test-helpers/src/setup.js:2
const BN = web3.utils.BN;
           ^

ReferenceError: web3 is not defined
```

Why web3 is not imported inside `setup.js` ? Am I missing something obvious ?

### PR DESCRIPTION
**Pull request by frangio**
Fixes #35 

Removes the assumption that there is a global web3 (as in a Truffle environment) by adding a mechanism to manually configure the web3 instance. Also allows using in Truffle migrations (https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1743).

```javascript
require('openzeppelin-test-helpers/configure')({ web3: ... });
```

Also where possible replaces `web3.utils` by the `web3-utils` package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,10 +2,11 @@
   "name": "openzeppelin-test-helpers",
   "version": "0.3.2",
   "description": "JavaScript testing helpers for Ethereum smart contract development.",
-  "main": "openzeppelin-test-helpers.js",
+  "main": "index.js",
```
Comment (nventuro) [inline package.json]: Ninja edit

File: package.json [hunk 2]
```diff
@@ -36,7 +37,8 @@
     "chai-bn": "^0.1.1",
     "ethjs-abi": "^0.2.1",
     "semver": "^5.6.0",
-    "truffle-contract": "^4.0.8"
+    "truffle-contract": "^4.0.8",
+    "web3-utils": "^1.0.0-beta.55"
```
Comment (nventuro) [inline package.json]: Is web3-utls versioned along web3? If so, I'd target beta.37, and check for that one.

File: inject-web3.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const { setWeb3 } = require('./src/global-web3');
+
+module.exports = function (web3) {
+  // [TODO] check version
```
Comment (nventuro) [inline inject-web3.js]: Isn't this done in `global-web3`?

File: package.json (hunk 2 above)
Comment (frangio) [reply package.json]: I don't really see why we should pin to beta 37, but perhaps it's a good idea just in case.

File: inject-web3.js (hunk 3 above)
Comment (frangio) [reply inject-web3.js]: Ah, yes, I forgot to remove this comment.

**Comment by frangio** [comment]
Sorry, let me add some documentation and a changelog entry.

**Comment by frangio** [comment]
@nventuro Done.

I also changed the API slightly. It is now called `configure` and receives an object. Additionally, it no longer returns the helpers, forcing you to get them as another step.

```
require('openzeppelin-test-helpers/configure')({ web3 });
const { expectEvent } = require('openzeppelin-test-helpers');
```

File: configure.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      loadedConfig = 'default';
+    }
+  } else {
+    if (loadedConfig) {
+      let errorMessage = 'Cannot configure openzeppelin-test-helpers a second time.';
+
+      if (loadedConfig === 'default') {
+        errorMessage += `
+
+A configuration has been loaded by default. Make sure to do custom configuration before importing the library.
```
Comment (nventuro) [inline configure.js]: :heart_eyes:

Comment (frangio) [reply configure.js]: :smile: I like the error messages. I don't like the source code in this file so much...

**Pull request merged**